Cubical/Relation/Binary/Base.agda

```diff
@@ -9,13 +9,13 @@ open import Cubical.HITs.SetQuotients.Base
 
 module BinaryRelation {ℓ ℓ' : Level} {A : Set ℓ} (R : A → A → Set ℓ') where
   isRefl : Set (ℓ-max ℓ ℓ')
-  isRefl = {a : A} → R a a
```

Comment:
What is the reason for changing these?

Reply:
Agda couldn't infer the argument in DiffInt.Properties so things got yellow there. I believe that we often want these arguments explicitly when doing these proofs, so I'd rather have them explicit and insert underscores sometimes.